capi/src/lib.rs

```diff
@@ -0,0 +1,195 @@
+#![allow(non_camel_case_types)]
+
+extern crate libc;
+extern crate telemetry;
+
+use std::ptr;
+use std::ffi::{CStr, CString};
+use std::mem;
+use std::str;
+use std::sync::mpsc::channel;
+use telemetry::{Subset, Service, SerializationFormat};
+use telemetry::plain::{Histogram, Flag, Count};
+
+pub struct telemetry_t {
+    service: Service,
+    flags: Vec<*mut flag_t>,
```

Comment:
Do I understand correctly that the role of `flags`/`counts` is solely to be able to drop `Flag`/`Count` histograms?